drivers/serial/uart_npcx.c

```diff
@@ -28,10 +28,17 @@ struct uart_npcx_config {
 	/* int-mux configuration */
 	const struct npcx_wui uart_rx_wui;
 	/* pinmux configuration */
-	const uint8_t   alts_size;
```

Comment:
Thanks for fixing this. Can you move the unrelated style changes to a separate commit?

Reply:
Ok. I have separated the coding style to the second commit.